olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/VersionedLayerClient.h

```diff
@@ -43,6 +43,11 @@ using DataResponseCallback = std::function<void(DataResponse response)>;
  */
 class DATASERVICE_READ_API VersionedLayerClient final {
  public:
+  /// ApiResponse template alias
+  using CallbackResponse = client::ApiResponse<model::Data, client::ApiError>;
```

Comment:
This is not really a template. The idea with the template was to not have an alias for every method.

Reply:
Aha, got your point. There are no other methods right now, but any way.